earthfile2llb/interpreter.go

```diff
@@ -1396,6 +1484,16 @@ func parseParans(str string) (string, []string, error) {
 	return parts[0], parts[1:], nil
 }
 
+func isSafeAsyncBuildArgs(args []string) bool {
+	for _, arg := range args {
+		_, v, _ := variables.ParseKeyValue(arg)
+		if strings.HasPrefix(v, "$(") {
```

Comment:
do we need to worry about backticks here too?

Reply:
No... the syntax for non-constant args only covers `$(...)`.